lms/static/styles/components/_ValidationMessage.scss

```diff
@@ -0,0 +1,46 @@
+@keyframes validationErrorOpen {
+  from {
+    width: 0;
+    opacity: 0;
+  }
+  to {
+    width: 300px;
+    opacity: 0.9;
+  }
+}
+
+@keyframes validationErrorClose {
+  from {
+    width: 300px;
+    opacity: 1;
+  }
+  to {
+    width: 0px;
+    opacity: 0;
+  }
+}
+
+.ValidationMessage {
```

Comment:
Nit: Can you please order these rules in the general order of:

1. Positioning/box model (display, z-index, position, etc.)
2. Layout (padding, margins, etc.)
3. Everything else